circle.yml

```diff
@@ -369,5 +375,6 @@ workflows:
             branches:
               only:
                 - master
+                - stage
```

Comment:
Cleanup workflow should only run on `master`, this is related to integration tests and automatically clears all dangling VMs and resources.

Reply:
@giovannicimolin Fare point. Reverting this part.